app/background/wallet/service.js

```diff
@@ -592,11 +586,12 @@ class WalletService {
     });
 
     node.wdb.client.bind('block connect', this.onNewBlock);
+    node.wdb.client.unhook('block rescan');
```

Comment:
How does this affect the CLI command `rescan` from API / hs-client? It might be ok to replace that endpoint with this custom GUI action, but we should be sure that if a user "accidentally" initiates a rescan from outside Bob (directly to the wallet node HTTP API)  that it doesn't leave the wallet in some weird state.